inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -202,6 +156,7 @@ void FrontEnd::detectNetworkBatch(
 
             _unbatchedOutputs.insert(ieData);
         }
+
         // preserve overriden output precision.
         ieData->setPrecision(outputsPresisions[p.first]);
```

Comment:
Looks suspecting. Do we need it? But I suggest taking a look at it during 48878. Just keep in mind